neqo-http3/tests/priority.rs

```diff
@@ -107,23 +107,20 @@ fn priority_update() {
     client.priority_update(stream_id, update_priority).unwrap();
     exchange_packets(&mut client, &mut server);
 
-    let priority_event = loop {
-        let event = server.next_event().unwrap();
-        if matches!(event, Http3ServerEvent::PriorityUpdate { .. }) {
-            break event;
-        }
-    };
-
-    match priority_event {
-        Http3ServerEvent::PriorityUpdate {
+    let found = server.events().any(|e| {
```

Comment:
Based on what I found in https://github.com/mozilla/neqo/issues/1304#issuecomment-1009787928, `server.events()` is empty here.